README.md

```diff
@@ -4,14 +4,18 @@
   <a href="https://github.com/ReactiveCircus/android-emulator-runner"><img alt="GitHub Actions status" src="https://github.com/ReactiveCircus/android-emulator-runner/workflows/Main%20workflow/badge.svg"></a>
 </p>
 
-A GitHub Action for installing, configuring and running hardware-accelerated Android Emulators on macOS virtual machines.
+A GitHub Action for installing, configuring and running Android Emulators on macOS and Ubuntu virtual machines.
```

Comment:
I think it's still important to distinguish between macOs and Ubuntu support. Being able to run in `swiftshader_indirect` mode is still way faster than `-accel off`. When people read this without any caveat they would expect running the action on linux VMs to be feasible, which is far from the reality.

Reply:
Addressed in commit: e860dce